tests/units/typing/url/test_any_url.py

```diff
@@ -18,3 +18,8 @@ def test_json_schema():
 def test_dump_json():
     url = parse_obj_as(AnyUrl, 'http://jina.ai/img.png')
     orjson_dumps(url)
+
+
+def test_relative_path():
+    # see issue: https://github.com/docarray/docarray/issues/978
+    parse_obj_as(AnyUrl, 'data/05978.jpg')
```

Comment:
can you test that when we get back the data it is still `data/05978.jpg` and not `file://data/05978.jpg` ?

Reply:
![image](https://user-images.githubusercontent.com/44071807/210375901-67ffe07a-3efe-4ea5-8ab2-02e5652461b6.png)
epic fail, good catch. working on it